core-bundle/src/Search/Escargot/Subscriber/EscargotSubscriber.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Search\Escargot\Subscriber;
+
+use Terminal42\Escargot\Subscriber\SubscriberInterface;
+
+interface EscargotSubscriber extends SubscriberInterface
```

Comment:
I just noticed that this does not follow our naming conventions. It should be `EscargotSubscriberInterface`.

Reply:
Oh, indeed! Good catch! Fixed in e591905a87c3c75dbfc378a980d4920bf595895a.